docs/guides/README.md

```diff
@@ -26,21 +26,23 @@ Kong Ingress controller:
   This guide shows how to expose services running outside Kubernetes via Kong,
   using [External Name](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
   Services in Kubernetes.
-- [Configuring HTTPS redirects for your services](configuring-https-redirect.md)  
+- [Configuring HTTPS redirects for your services](configuring-https-redirect.md)
   This guide walks through how to configure Kong Ingress Controller to
   redirect HTTP request to HTTPS so that all communication
   from the external world to your APIs and microservices is encrypted.
-- [Using Redis for rate-limiting](redis-rate-limiting.md)  
+- [Using Redis for rate-limiting](redis-rate-limiting.md)
   This guide walks through how to use Redis for storing rate-limit information
   in a multi-node Kong deployment.
-- [Integrate Kong Ingress Controller with Prometheus/Grafana](prometheus-grafana.md)  
+- [Integrate Kong Ingress Controller with Prometheus/Grafana](prometheus-grafana.md)
   This guide walks through the steps of how to deploy Kong Ingress Controller
   and Prometheus to obtain metrics for the traffic flowing into your
   Kubernetes cluster.
-- [Configuring circuit-breaker and health-checking](configuring-health-checks.md)  
+- [Configuring circuit-breaker and health-checking](configuring-health-checks.md)
```

Comment:
These spaces are intentional. They add a `\n` to the markdown rendering. If you see this vs master markdown rendering, you will see that the description of the guide is now is not on a newline anymore. Please revert this change.